tests/installation/partitioning.pm

```diff
@@ -36,8 +36,7 @@ sub run {
             $cmd{rescandevices}     = 'alt-c';
             $cmd{enablelvm}         = 'alt-a';
             $cmd{encryptdisk}       = 'alt-l';
-            # On TW have different shortcut for toggling separate home partition
-            $cmd{toggle_home} = 'alt-o' unless is_leap;
+            $cmd{toggle_home}       = 'alt-p';
```

Comment:
We have this already defined in susedistribution:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L81